frontend/src/utils/misc.js

```diff
@@ -0,0 +1,6 @@
+export function trimUsername(username, length) {
```

Comment:
For consistency, could we remove `export` here and add `export {trim username}` to the bottom of the file?